components/kucoin_futures/actions/generic-api-call/generic-api-call.mjs

```diff
@@ -0,0 +1,51 @@
+import kucoin_futures from "../../kucoin_futures.app.mjs";
+
+export default {
+  name: "Kucoin Futures Generic API Call",
+  version: "0.0.1",
+  key: "kucoin_futures-generic-api-call",
+  description: "Make any API call for Kucoin Futures as per the documentation." +
+        "[reference](https://docs.kucoin.com/futures/#general)",
+  props: {
+    kucoin_futures,
+    http_method: {
+      label: "HTTP Method",
+      description: "HTTP Method as per API documentation",
+      type: "string",
+      optional: false,
+      options: [
+        "GET",
+        "DELETE",
+        "POST",
+        "PUT",
+      ],
+    },
+    api_path: {
+      label: "API Path",
+      description: "Ex. /api/v1/orders",
+      type: "string",
+      optional: false,
+    },
+    params: {
+      label: "Parameters",
+      description: "API Parameters",
+      type: "object",
+      optional: true,
+    },
+  },
+  type: "action",
+  async run({ $ }) {
+    if (!this.api_path.startsWith("/")) {
+      throw new Error("Api Path property should always start with `/` as a path like string");
```

Comment:
Please throw a ConfigurationError on this case

[see docs here:](https://pipedream.com/docs/code/nodejs/#configuration-error) 
`Throwing a ConfigurationError in a Node.js step will display the error message in a dedicated area.
This is useful for providing feedback during validation of props. In the example below, a required Header value is missing from the Google Sheets action:`